int32 car_num
float64 x
float64 y
float64 speed
